src/main/java/org/jboss/hal/testsuite/page/config/ConfigurationPage.java

```diff
@@ -21,7 +23,7 @@
     public ConfigurationPage select(String label) {
         String cellClass = PropUtils.get("table.cell.class");
         String cellSelectedClass = PropUtils.get("table.cell.selected.class");
-        By selector = ByJQuery.selector("." + cellClass + ":contains('" + label + "')");
+        By selector = By.ByXPath.xpath("//td[contains(@class,'" + cellClass + "') and descendant::div[@class='navigation-column-item' and text()='" + label + "']]");
```

Comment:
Wouldn't it be possible to use rather jquery even for equals? I did quick google search and found this: https://api.jquery.com/attribute-equals-selector/